### PR TITLE
updates links and descriptions for gigabit portfolio items

### LINKED ISSUE #2514: Add links / update descriptions of Gigabit portfolio items
Per this [spreadsheet](https://docs.google.com/spreadsheets/d/1Hd6DtgQpUd2rqCtI5TQIdHmvLlgOGa6Whf5BdHHUslM/edit#gid=1550807061)

### PR DESCRIPTION
**Pull request by hannahkane**
Fixes #2514

**Changes proposed in this pull request:**
- updates a bunch of links and descriptions per #2514

### CODE REVIEW

**Comment by hannahkane** [comment]
cc @jenntbeard - filing a pull request for the last of the changes, but since I didn't get to this until late in the day, it won't go live until tomorrow at the earliest (pending review from an engineer)

**Comment by hannahkane** [comment]
@gvn - can you help me parse the reason for the build fail?

File: pages/gigabit/portfolio.json [hunk 1]
```diff
@@ -782,6 +908,10 @@
       {
         "url": "http://www.globalkids.org/",
         "label": null
+      },
+      {
+        "url": "sites.google.com/globalkids.org/hauntspd/",
```
Comment (gvn) [inline pages/gigabit/portfolio.json]: This needs a `https://` prefix ^

**Comment by hannahkane** [comment]
Ah, thanks, @gvn! How's that?

File: pages/gigabit/portfolio.json [hunk 2]
```diff
@@ -523,6 +573,18 @@
       {
         "url": "http://www.austinmonitor.com/",
         "label": null
+      },
+      {
+        "url": "http://austinbudget.party/home",
+        "label": "App"
+      },
+      {
+        "url": "github.com/open-austin/budgetparty",
```
Comment (gvn) [inline pages/gigabit/portfolio.json]: Missing protocol ^

**Comment by hannahkane** [comment]
sheesh. sorry I'm so bad at this. how about now?

**Comment by gvn** [comment]
Looks good!

**Pull request merged**